znnl/training_strategies/simple_training.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         # Add the loss and accuracy function to the recorders and re-instantiate them
```

Comment:
adapt the comment

Reply:
I'm not sure what you mean, could you explain this further?

Reply:
I mean you now also include the model there.

Reply:
Was fixed in commit [d17e514](https://github.com/zincware/ZnNL/pull/109/commits/d17e5149e0e93df1ee89509ff16fca738950f430)